src/main/java/com/faforever/client/preferences/PreferencesService.java

```diff
@@ -175,6 +175,14 @@ public Path getFafDataDirectory() {
     return FAF_DATA_DIRECTORY;
   }
 
+  public Path getSecondaryFallbackVaultLocation() {
+    return Paths.get(FAF_DATA_DIRECTORY.toAbsolutePath().toString(), "user", "My Games", "Gas Powered Games", "Supreme Commander Forged Alliance");
+  }
```

Comment:
Is this a secondary vault location or a fallback vault location?
And if it's a secondary fallback location, then the secondary fallback or the fallback of the secondary location? And what about a terti....

Reply:
Secondary is similar to fall back. You mean I should decide on one or what????

Reply:
Renamed to only Secondary...